docs/zh/react/reactlynx-testing-library/index.mdx

```diff
@@ -181,6 +104,8 @@ it('basic render', () => {
 
 可以直接自己构造 `Event` 对象，也可以使用直接传入事件类型和初始化参数让 Testing Library 自动构造 `Event` 对象。
```

Comment:
Enumerating all `eventType` as markdown bullet list is pretty bad... consider defining it in a more formal manner (e.g. formal syntax, API reference)